Mathlib/Util/AddRelatedDecl.lean

```diff
@@ -42,14 +42,15 @@ Arguments:
 -/
 def addRelatedDecl (src : Name) (suffix : String) (ref : Syntax)
     (attrs? : Option (Syntax.TSepArray `Lean.Parser.Term.attrInstance ","))
-    (construct : Expr → Expr → List Name → MetaM (Expr × List Name)) :
+    (construct : Expr → List Name → MetaM (Expr × List Name)) :
```

Comment:
The above doc comment should be updated for the new type of the `construct` parameter.